Makefile

```diff
@@ -50,8 +50,12 @@ build:
 	go build -ldflags "-s -w -X=github.com/algolia/cli/pkg/version.Version=$(VERSION)" -o algolia cmd/algolia/main.go
 .PHONY: build
 
-## Install & uninstall tasks are here for use on *nix platform only.
+# Generate flags for commands
```

Comment:
I don't think this task is necessary, since you can simply do `go generate ./...`.
If you look at the `build` phase above, it's already included.

Reply:
Got it! I removed it 🙏